apics/static/Atlas/95.html

```diff
@@ -401,7 +401,7 @@ <h4 class="c11" id="section-2thevalues"><span class="c7">2. The values</span><a
 </div>
 </div>
 </div>
-<p class="western c22"><span class="c10">In San Andres Creole, the
+<p class="western c22"><span class="c10"><a href="/contributions/10">San Andres Creole</a>, the
```

Comment:
Where did the "In" go?

Reply:
Thank you for your attentiveness :) now it is back in its place

Reply:
Ok, but putting the fix in a separate pull request means I don't know what to do with this one. The first change above would also need to go in a new PR so I can close this one.